src/backend/configs/apiv2.yaml

```diff
@@ -18,7 +18,7 @@ ApiConfig:
         buckets:
           raw_messages: caliopen-raw-messages                # bucket name to put raw messages to
           temporary_attachments: caliopen-tmp-attachments    # bucket name to store draft attachments
-      use_vault: false
+      use_vault: true
```

Comment:
Can't be set to true for development platform, credentials are in stored in user identity on such platform

Reply:
my mistake, it was for testing purpose and I forgot to revert…